demos/prefsFramework/index.html

```diff
@@ -384,7 +386,7 @@ <h1 class="ocean">Climate Change Impacts</h1>
                             </div>
                             <div class="right_tab"></div><div style="clear:both;"></div>
                             <div id="newsReleasesList" style="overflow: hidden;border: 1px #CCC solid ;margin-bottom: 11px; ">
-                                <h3 style="padding:5px; padding-bottom:0px; background-color:#F0F8FF">Multimedia</h3>
+                                <h2 style="padding:5px; padding-bottom:0px; background-color:#F0F8FF">Multimedia</h2>
```

Comment:
I wonder if it makes sense to leave these as `<h3>` and make the "Resources" text above an `<h2>`?